port/unix/omrsl.c

```diff
@@ -438,3 +438,97 @@ omrsl_startup(struct OMRPortLibrary *portLibrary)
 {
 	return 0;
 }
+
+/**
+ * Native Libraries
+ *
+ * This function is called go get all the shared libraries loaded by a process.
+ *
+ * @param[in] portLibrary Pointer to the OMR port library.
+ * @param[in] callback Function to be called for each library found.
+ * @param[in] userData User-defined data passed to the callback.
+ *
+ * @return 0 if successful, or the first non-zero return value from the callback.
+ */
+uintptr_t
+omrsl_get_libraries(struct OMRPortLibrary *portLibrary, OMRLibraryInfoCallback callback, void *userData)
+{
+#if defined(LINUX)
+	#include <limits.h>
```

Comment:
This isn't likely to work (within the function body); please move it near the top of the file (if needed).